internal/postgresql/postgresql.go

```diff
@@ -306,8 +308,10 @@ func (p *Manager) start(args ...string) error {
 	// the instance parent is the keeper instead of the defined system reaper
 	// (since pg_ctl forks and then exits leaving the postmaster orphaned).
 
-	if err := p.createPostgresqlAutoConf(); err != nil {
-		return err
+	if p.disableAlterSystem {
```

Comment:
We should add some code to manage the removal of the symlink if someone was previously running with `disable-alter-system=true`, and now wants to run with it enabled.

In the current form, if Stolon users were to upgrade to a Stolon version with this PR, they'd have lots of `/dev/null` symlink files remaining on their machines. I think it's best if we can detect when the symlink to `/dev/null` is there and replace it with a blank `postgresql.auto.conf` file when we've got alter system enabled, as this would be least surprising to me as a user.

Reply:
ok, added this behaviour just right now